### DISCUSSION TITLE
Cannot add new PostgreSQL Source, cannot Retest saved source and cannot Refresh source schema after upgrade to 0.50.34

### DISCUSSION THREAD

**Discussion by rcnavas** [airbytehq/airbyte#32714] [Connector Questions]
I upgraded Airbyte to version: 0.50.34
Using Source Postgresql version: 3.2.21
Using Destination Bigquery version: 2.3.16
After the upgrade, I have 3 issues:

1. Trying to "Retest saved source" gives error "Configuration Check failed" `Internal message: scheduledEventId=6, startedEventId=7, activityType='RunWithJobOutput', activityId='83311e58-d45e-3bda-a6be-408deb2498fc', identity='1@75e5ff5a6760', retryState=RETRY_STATE_MAXIMUM_ATTEMPTS_REACHED`
2. Trying to add a New Source using same Postgresql connector also give the same error above.
3. Trying to "Refresh Source schema" gives error: `Internal Server Error: Cannot invoke "io.airbyte.api.model.generated.AirbyteCatalog.getStreams()" because "discovered" is null`
<img width="564" alt="Captura de pantalla 2023-11-21 a la(s) 15 13 22" src="https://github.com/airbytehq/airbyte/assets/4992755/6b8d66c1-4b8e-4ddf-944b-814ef881b759">
<img width="1012" alt="Captura de pantalla 2023-11-21 a la(s) 15 13 41" src="https://github.com/airbytehq/airbyte/assets/4992755/1c1acde0-d5f6-4e71-b68f-f15b1638d868">

**Comment by marcosmarxm** [comment]
Hello @rcnavas are you still having the issue?

[a comment by an automated account is not shown]
